velox/expression/VectorUdfTypeSystem.h

```diff
@@ -370,11 +376,12 @@ struct VectorWriter<Array<V>> {
   void ensureSize(size_t size) {
     // todo(youknowjack): optimize the excessive ensureSize calls
     if (size > vector_->size()) {
-      vector_->resize(size);
+      vector_->resize(size, /*setNotNull*/ false);
       init(*vector_);
     }
   }
 
+  // TODO: deprecate this once all proxies introduced.
```

Comment:
nit: I would rather you create a task to remind you about this, then have this todo, here even if you will do it next pr.

Reply:
I will create a task, this comment os on different PR haha, github sucks